### PR TITLE
improve consumer import

### PR DESCRIPTION
**Pull request by maxarndt**
Should address #69

### CODE REVIEW

File: jetstream/resource_jetstream_consumer.go [hunk 1]
```diff
@@ -480,11 +480,11 @@ func resourceConsumerRead(d *schema.ResourceData, m any) error {
...
 	switch cons.DeliverPolicy() {
 	case api.DeliverAll:
-		d.Set("delivery_all", true)
+		d.Set("deliver_all", true)
 	case api.DeliverNew:
-		d.Set("delivery_new", true)
+		d.Set("deliver_new", true)
 	case api.DeliverLast:
-		d.Set("delivery_last", true)
+		d.Set("deliver_last", true)
```
Comment (ripienaar) [inline jetstream/resource_jetstream_consumer.go:487]: doh, well spotted.

**Review by ripienaar — COMMENTED** [review]
this looks good to me, cant believe its down to typos :( thank you, let me know when you're ready to merge.

File: jetstream/resource_jetstream_consumer.go (hunk 1 above)
Comment (ripienaar) [reply jetstream/resource_jetstream_consumer.go:487]: Hmm, I think these are in fact `all`, `last`, `by_start_sequence`, `by_start_time`, `last_per_subject` and `new`

Comment (ripienaar) [reply jetstream/resource_jetstream_consumer.go:487]: Would also need to fix all these 

https://github.com/nats-io/terraform-provider-jetstream/blob/b43dbe2e9c54fb27881066a1920af0eff70745c9/jetstream/resource_jetstream_consumer.go#L72

Just take those lines out on the deliver policy ones as tehy are all boolean

**Comment by ripienaar** [comment]
Thank you!

**Comment by rayjanoka** [comment]
thanks!

**Pull request merged**